src/main/kotlin/no/nav/familie/ks/sak/api/dto/BehandlingStegDto.kt

```diff
@@ -68,3 +69,10 @@ enum class HenleggÅrsak(val beskrivelse: String) {
         TEKNISK_VEDLIKEHOLD -> Behandlingsresultat.HENLAGT_TEKNISK_VEDLIKEHOLD
     }
 }
+
+data class TilbakekrevingDto(
```

Comment:
Skal vi splitte `TilbakekrevingDto` og `TilbakekrevingResponsDto` ut til egen fil?
Nå ligger de spredt utover BehandlingStegDto.kt og BehandlingResponsDto.kt.

Stemmer også for å rename TilbakekrevingDto -> TilbakekrevingRequestDto

Reply:
@UyQuangNguyen TilbakekrevingDto  kan kalles TilbakekrevingRequestDto. Men jeg tror vi ikke trenger å splitte det til egen fil. TilbakekrevingRequestDto arver fra BehandlingStegDto. Så det gir mening å ligge den dto-en i BehandingStegDto. BehandlingResponsDto returnerer tilbakekreving så det er greit å ha TilbakekrevingResponsDto der.

Reply:
Ja, det høres bra ut.
Bare å ta renaminga så er jeg fornøyd 🤩